lib/generators/suspenders/jobs_generator.rb

```diff
@@ -0,0 +1,55 @@
+module Suspenders
+  module Generators
+    class JobsGenerator < Rails::Generators::Base
+      include Suspenders::Generators::APIAppUnsupported
+      source_root File.expand_path("../../templates/active_job", __FILE__)
+      desc "Installs sidekiq for background job processing."
+
+      def add_sidekiq_gem
+        gem "sidekiq"
+        Bundler.with_unbundled_env { run "bundle install" }
+      end
+
+      def initialize_active_job
+        copy_file "active_job.rb", "config/initializers/active_job.rb"
+      end
+
+      def configure_active_job
+        configure_application_file("config.active_job.queue_adapter = :sidekiq")
+        configure_application_file("config.action_mailer.deliver_later_queue_name = nil")
+        configure_application_file("config.action_mailbox.queues.routing = nil")
+        configure_application_file("config.active_storage.queues.analysis = nil")
+        configure_application_file("config.active_storage.queues.purge = nil")
+        configure_application_file("config.active_storage.queues.mirror = nil")
+        configure_environment "test", "config.active_job.queue_adapter = :inline"
```

Comment:
I think we can leverage the [environment](https://guides.rubyonrails.org/rails_application_templates.html#environment-application-data-nil-options-block) command here, which would alleviate us from having to create a custom method.